packages/blade/src/components/InfoGroup/_decisions/decisions.md

```diff
@@ -0,0 +1,333 @@
+# InfoGroup
+
+InfoGroup is a structured component for displaying key-value pairs in a consistent, organized format. It provides a standardized way to present information such as transaction details, user data, or any related data pairs with proper visual hierarchy and alignment. The component supports various orientations, sizes, and customization options to suit different use cases across dashboards, modals, drawers, and detail views.
+
+<p align="center"><img src="./2025-05-28-13-36-53.png" alt="InfoGroup" width="800px" /></p>
+
+## Design
+
+- [Figma - InfoGroup](https://www.figma.com/design/jubmQL9Z8V7881ayUD95ps/Blade-DSL?node-id=99000-73547&p=f&m=dev)
+
+## API
+
+The InfoGroup component uses a compound structure with an outer container and InfoItem sub-components for individual key-value pairs. This approach allows layout and styling props to be defined once at the container level.
+
+```jsx
+import {
+  InfoGroup,
+  InfoItem,
+  InfoItemKey,
+  InfoItemValue,
+  Divider,
+} from '@razorpay/blade/components';
+
+<InfoGroup itemOrientation="horizontal" size="medium">
+  <InfoItem>
+    <InfoItemKey leading={UserIcon} helpText="Customer information">
+      Account Holder
+    </InfoItemKey>
+    <InfoItemValue trailing={CheckIcon}>Saurabh Daware</InfoItemValue>
+  </InfoItem>
+  <Divider />
+  <InfoItem>
+    <InfoItemKey>Payment ID</InfoItemKey>
+    <InfoItemValue trailing={CopyIcon}>
+      <Code size="small">pay_MK7DGqwYXEwx9Q</Code>
+    </InfoItemValue>
+  </InfoItem>
+</InfoGroup>;
+```
+
+<details>
+  <summary>Alternate APIs</summary>
+
+### Alternate API 1: Single Component with Props
+
+```jsx
+<InfoGroup itemOrientation="horizontal" size="medium">
+  <InfoItem
+    keyText="Account Holder"
+    valueText="Saurabh Daware"
+    keyIcon={UserIcon}
+    valueIcon={CheckIcon}
+  />
+  <InfoItem keyText="Payment ID" valueText="pay_MK7DGqwYXEwx9Q" valueType="code" />
+</InfoGroup>
+```
+
+**Pros:**
+
+- Simpler API with fewer components to remember
+- Faster to write for basic use cases
+- Consistent prop naming pattern
+- Less verbose for simple key-value pairs
+
+**Cons:**
+
+- Less flexible for complex value layouts
+- Cannot compose with other Blade components easily
+- Limited customization options for styling individual parts
+- Props can become numerous for complex scenarios
+- Harder to extend for future use cases
+
+</details>
+
+### Props
+
+#### InfoGroup
```

Comment:
Add styled props in InfoGroup so we can control margins etc. Maybe skipping flex related props.